### PR TITLE
Translate 10 common pages in Italian

### PR DESCRIPTION
**Pull request by gRastello**
Translated pages for some useful common tools.

`ls cd cp rm mv mkdir tac head vim less ln`

### CODE REVIEW

**Comment by mebeim** [comment]
Hi @gRastello, thank you for your contribution!

I see you have translated the `rm` command, but the commit message is wrong, it says: `cd: add Italian translation.`, could you please reword it? Also, you can squash [`2592e8`](https://github.com/gRastello/tldr/commit/2592e85bef9390b21bc3fd38192c8c986685b592) and [`91dc91`](https://github.com/gRastello/tldr/commit/91dc9151a7f27f4180411a7f2530e662d5faedd5) together if you want, no need to have two separate commits since the page is new. If you are unsure about how to do it, let me know and I can guide you.

File: pages.it/common/cp.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# cp
+
+> Copia files e directory
+
+- Copia un file in un altro file:
```
Comment (mebeim) [inline pages.it/common/cp.md]:
```suggestion
- Copia un file in un'altra posizione:
```

File: pages.it/common/cp.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`cp {{percorso/al/file}} {{percorso/alla/copia}}`
+
+- Copia un file in una directory mantenendo il nome:
+
+`cp {{percorso/al/file}} {{percorso/alla/directory}}`
+
+- Copia una directory ricorsivamente in un altro posto:
```
Comment (mebeim) [inline pages.it/common/cp.md]:
```suggestion
- Copia una directory ricorsivamente in un'altra posizione:
```

File: pages.it/common/cp.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`cp -r {{percorso/alla/directory}} {{percorso/alla/copia}}`
+
+- Copia una directory ricorsivamente in modo verboso (mostra a schermo ogni volta che un file viene copiato):
```
Comment (mebeim) [inline pages.it/common/cp.md]:
```suggestion
- Copia una directory ricorsivamente in modo verboso (mostra a schermo ogni file copiato):
```

File: pages.it/common/cp.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# cp
+
+> Copia files e directory
```
Comment (mebeim) [inline pages.it/common/cp.md]:
```suggestion
> Copia file e directory.
```

The terminal -s is not used for English plurals in Italian. I see you used this form extensively, so I'm only commenting this particular case to not repeat myself, but please **apply the correction to any other English plural in other pages too** (e.g. `bytes`).

File: pages.it/common/cp.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- Copia una directory ricorsivamente in modo verboso (mostra a schermo ogni volta che un file viene copiato):
+
+`cp -vr {{percorso/alla/directory}} {{percorso/alla/copia}}`
+
+- Copia i contenuti di una directory in una seconda directory:
+
+`cp -r {{percorso/alla/directory/*}} {{percorso/alla/seconda/directory}}`
+
+- Copia tutti i txt files in una directory in modo interattivo, chiedendo conferma prima di copiare ogni file:
```
Comment (mebeim) [inline pages.it/common/cp.md]:
```suggestion
- Copia tutti i file di testo in una seconda directory in modo interattivo (chiede conferma prima di sovrascrivere):
```

The `-i` option only asks before overwriting, not for any file.

File: pages.it/common/less.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# less
+
+> Apri un file per lettura interattiva.
```
Comment (mebeim) [inline pages.it/common/less.md]:
```suggestion
> Apri un file per lettura interattiva, permettendo di scorrere e cercare testo al suo interno.
```

Let's be more verbose here following the English description.

File: pages.it/common/less.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+- Apri un file:
+
+`less {{file}}`
+
+- Muoviti una pagina giù / su:
```
Comment (mebeim) [inline pages.it/common/less.md]:
```suggestion
- Muoviti una pagina in giù / su:
```

File: pages.it/common/less.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+`<Space>, b`
```
Comment (mebeim) [inline pages.it/common/less.md]:
```suggestion
`<Spazio> (giù), b (su)`
```

File: pages.it/common/less.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+- Vai all'inizio o alla fine del file:
+
+`G, g`
```
Comment (mebeim) [inline pages.it/common/less.md]:
```suggestion
`G (fine), g (inizio)`
```

File: pages.it/common/ls.md    [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+`ls -lh`
+
+- Elenca tutti i file e mostra informazioni, ordinandoli per dimensione decrescente:
+
+`ls -lS`
+
+- Elenca tutti i file e mostra informazioni, ordinandoli per ultima modifica:
+
+`ls -ltr`
```
Comment (mebeim) [inline pages.it/common/ls.md   ]: Missing a trailing newline in this file.

File: pages.it/common/ln.md [hunk 11]
```diff
@@ -0,0 +1,15 @@
+# ln
+
+> Crea un collegamento a un file o a una directory.
+
+- Crea un collegamento simbolico a un file (o directory):
+
+`ln -s {{percorso/al/file}} {{percorso/al/collegamento}}`
+
+- Sovrascrivi un esistente collegamento in modo che punti a un nuovo file:
```
Comment (mebeim) [inline pages.it/common/ln.md]:
```suggestion
- Sovrascrivi un collegamento esistente in modo che punti a un nuovo file:
```

File: pages.it/common/ls.md    [hunk 12]
```diff
@@ -0,0 +1,27 @@
+# ls
+
+> Elenca i contenuti di una directory.
+
+- Elenca i file nella directory, uno riga per riga:
```
Comment (mebeim) [inline pages.it/common/ls.md   ]:
```suggestion
- Elenca i file nella directory corrente, uno per riga:
```

File: pages.it/common/mkdir.md [hunk 13]
```diff
@@ -0,0 +1,11 @@
+# mkdir
+
+> Crea directory.
+
+- Crea una directory nella directory corrente o in un dato percorso:
+
+`mkdir {{directory}}`
+
+- Crea directory ricorsivamente (utile per creare alberi di directory):
```
Comment (mebeim) [inline pages.it/common/mkdir.md]:
```suggestion
- Crea directory ricorsivamente (utile per creare directory annidate):
```

File: pages.it/common/rm.md   [hunk 14]
```diff
@@ -0,0 +1,23 @@
+# rm
+
+> Rimuovi files o directory.
+
+- Rimuovi files:
+
+`rm {{percorso/a/un/file}} {{percorso/a/un/secondo/file}}`
```
Comment (mebeim) [inline pages.it/common/rm.md  ]:
```suggestion
`rm {{percorso/a/file1 percorso/a/file2 …}}`
```
Let's simplify here.

File: pages.it/common/rm.md   [hunk 15]
```diff
@@ -0,0 +1,23 @@
...
+
+- Rimuovi ricorsivamente una directory (rimuovi una directory e tutti i suoi contenuti):
```
Comment (mebeim) [inline pages.it/common/rm.md  ]:
```suggestion
- Rimuovi ricorsivamente una directory e tutti i suoi contenuti:
```

No need for repetition.

File: pages.it/common/rm.md   [hunk 16]
```diff
@@ -0,0 +1,23 @@
...
+
+`rm -rf {{percorso/alla/directory}}`
+
+- Rimuovi files interattivamente, chiedendo conferma prima di rimuovere ogni file:
+
+`rm -i {{file(s)}}`
+
+- Rimuovi files in modalità verbosa, scrivendo un messaggio a schermo per ogni file rimosso:
+
+`rm -v {{percorso/a/un/file}}`
```
Comment (mebeim) [inline pages.it/common/rm.md  ]: Missing trailing newline.

File: pages.it/common/tac.md [hunk 17]
```diff
@@ -0,0 +1,11 @@
+# tac
+
+> Stampa e concatena files a partire dal fondo.
```
Comment (mebeim) [inline pages.it/common/tac.md]:
```suggestion
> Stampa e concatena files a al contrario.
```

File: pages.it/common/vim.md [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+
+- Apri un file in vim:
+
+`vim {{file}}`
+
+- Vai in insert mode (per inserire testo):
+
+`<Esc>i`
+
+- Copia o taglia la linea corrente (per incollarla poi con `P`):
```
Comment (mebeim) [inline pages.it/common/vim.md]:
```suggestion
- Copia ("yank") o taglia ("delete") la linea corrente (per incollarla poi con `P`):
```
I think it's better leave the meaning of the commands to be more clear.

File: pages.it/common/vim.md [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+
+`<Esc>{{yy|dd}}`
+
+- Annulla l'ultima operazione:
+
+`<Esc>u`
+
+- Cerca per un pattern nel file (usa `n`/`N` per spostarti al prossimo/precedente risultato):
```
Comment (mebeim) [inline pages.it/common/vim.md]:
```suggestion
- Cerca un pattern nel file (usa `n`/`N` per spostarti al risultato successivo/precedente):
```

Consistent with the same line in the `less` command.

File: pages.it/common/less.md [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+
+- Cerca in avanti per una stringa (usa `n`/`N` per muoverti al risultato successivo/precedente):
```
Comment (mebeim) [inline pages.it/common/less.md]:
```suggestion
- Cerca una stringa in avanti (usa `n`/`N` per muoverti al risultato successivo/precedente):
```

File: pages.it/common/less.md [hunk 21]
```diff
@@ -0,0 +1,35 @@
...
+
+`/{{stinga}}`
+
+- Cerca in indietro per una stringa (usa `n`/`N` per muoverti al risultato successivo/precedente):
```
Comment (mebeim) [inline pages.it/common/less.md]:
```suggestion
- Cerca una stringa indietro (usa `n`/`N` per muoverti al risultato successivo/precedente):
```

No need to say "per".

File: pages.it/common/less.md [hunk 22]
```diff
@@ -0,0 +1,35 @@
...
+
+`/{{stringa}}`
```
Comment (mebeim) [inline pages.it/common/less.md]:
```suggestion
`?{{stringa}}`
```
Backward search is done with `?`, not `/`.

File: pages.it/common/less.md (hunk 20 above)
Comment (mebeim) [reply pages.it/common/less.md]: No need to say "per".

File: pages.it/common/vim.md [hunk 23]
```diff
@@ -0,0 +1,36 @@
...
+
+`<Esc>/{{pattern}}<Enter>`
+
+- Effettua una sostituzione con regex sull'intero file:
```
Comment (mebeim) [inline pages.it/common/vim.md]:
```suggestion
- Effettua una sostituzione tramite regex nell'intero file:
```

File: pages.it/common/vim.md [hunk 24]
```diff
@@ -0,0 +1,36 @@
...
+
+`<Esc>:%s\{{pattern}}/{{sostituzione}}/g<Enter>`
```
Comment (mebeim) [inline pages.it/common/vim.md]:
```suggestion
`<Esc>:%s/{{pattern}}/{{sostituzione}}/g<Invio>`
```
Translate "Enter" too. Also a typo.

File: pages.it/common/vim.md [hunk 25]
```diff
@@ -0,0 +1,36 @@
...
+
+`<Esc>{{yy|dd}}`
+
+- Annulla l'ultima operazione:
+
+`<Esc>u`
+
+- Cerca per un pattern nel file (usa `n`/`N` per spostarti al prossimo/precedente risultato):
+
+`<Esc>/{{pattern}}<Enter>`
```
Comment (mebeim) [inline pages.it/common/vim.md]:
```suggestion
`<Esc>/{{pattern}}<Invio>`
```
Translate "Enter" too.

File: pages.it/common/vim.md [hunk 26]
```diff
@@ -0,0 +1,36 @@
...
+
+- Effettua una sostituzione con regex sull'intero file:
+
+`<Esc>:%s\{{pattern}}/{{sostituzione}}/g<Enter>`
+
+- Salva modifiche al file ed esci:
+
+`<Esc>:wq<Enter>`
```
Comment (mebeim) [inline pages.it/common/vim.md]:
```suggestion
`<Esc>:wq<Invio>`
```

File: pages.it/common/vim.md [hunk 27]
```diff
@@ -0,0 +1,36 @@
...
+
+- Esci senza salvare:
+
+`<Esc>:q!<Enter>`
```
Comment (mebeim) [inline pages.it/common/vim.md]:
```suggestion
`<Esc>:q!<Invio>`
```

**Comment by gRastello** [comment]
Thank you too @mebeim for all the suggestions and stuff. I may find some time to translate more pages in the future since I really like the project.

**Comment by mebeim** [comment]
@gRastello you're welcome, that's great, looking forward to it.

**Pull request merged**